### ISSUE TITLE
Feature Request: (>>>>>)

### ISSUE THREAD

**Issue by andys8** [Fresheyeball/elm-function-extra#2]
I'd need `(>>>>>)`. Would you add it or accept a pull request and release new version?

#### Example implementation
```elm
(>>>>>) : (a -> b -> c -> d -> e) -> (e -> f) -> a -> b -> c -> d -> f
(>>>>>) ffff f w x y z =
    ffff w x y z |> f
```

Hint: flipped version is necessary, too.

**Comment by cobalamin** [comment]
I'm not the maintainer, but I think that from a library perspective, this would be a bit silly to include. It's a very specific function for a specific parameter count of four, and the library doesn't offer `(>>>)` or `(>>>>)` so there's not much reason to add `(>>>>>)`.

Besides, this kind of function can be created easily and for arbitrary lengths (output from elm-repl):

```
> (<<)<<(<<)<<(<<)<<(<<) |> flip
<function> : (a -> a1 -> a2 -> a3 -> b) -> (b -> c) -> a -> a1 -> a2 -> a3 -> c
> (<<)<<(<<)<<(<<)<<(<<)
<function> : (b -> c) -> (a -> a1 -> a2 -> a3 -> b) -> a -> a1 -> a2 -> a3 -> c
````

**Comment by andys8** [comment]
The library doesn't expose `(>>>)` or `(>>>>)`? I either don't understand the point or this is wrong.

http://package.elm-lang.org/packages/Fresheyeball/elm-function-extra/3.0.0/Function

https://github.com/Fresheyeball/elm-function-extra/blob/master/Function.elm

**Comment by cobalamin** [comment]
You're right, sorry. Disregard that, I checked in the wrong place. The real point is that:

- (sadly) this isn't abstractable to an arbitrary amount of parameters in a nice way, so libraries have to draw the line at some arbitrary amount of parameters, for instance, core stops the class of `map`-like functions at `map5` (with a reason given: http://package.elm-lang.org/packages/elm-lang/core/5.1.1/List#map5).
- (but luckily) this function is very straightforward to write yourself once, as I mentioned above.
